pkg/core/config/tenant.go

```diff
@@ -35,6 +36,14 @@ type UserAuditConfiguration struct {
 	TrailHandlerURL string `msg:"TRAIL_HANDLER_URL" envconfig:"USER_AUDIT_TRAIL_HANDLER_URL" json:"TRAIL_HANDLER_URL"`
 }
 
+type SSOConfiguration struct {
+	Name         string `msg:"NAME" envconfig:"SSO_CONFIG_NAME" json:"NAME"`
+	Enabled      bool   `msg:"ENABLED" envconfig:"SSO_CONFIG_ENABLED" json:"ENABLED"`
+	ClientID     string `msg:"CLIENT_ID" envconfig:"SSO_CONFIG_CLIENT_ID" json:"CLIENT_ID"`
+	ClientSecret string `msg:"CLIENT_SECRET" envconfig:"SSO_CONFIG_CLIENT_SECRET" json:"CLIENT_SECRET"`
+	Scope        string `msg:"SCOPE" envconfig:"SSO_CONFIG_SCOPE" json:"SCOPE"`
```

Comment:
The overall direction of PR looks fine. But for the sso config, can we provide different config based on providers by environment variable? It would be better if we can have .env.example for auth gear too, so that we can set this up easily for development.